core/compiler.cpp

```diff
@@ -173,6 +173,20 @@ torch::jit::script::Module CompileGraph(const torch::jit::script::Module& mod, C
   return new_mod;
 }
 
+torch::jit::script::Module EmbedEngineInNewModule(const std::string& engine) {
+  std::ostringstream engine_id;
+  engine_id << reinterpret_cast<const int*>(&engine);
+  torch::jit::script::Module new_mod("tensorrt_engine_mod_" + engine_id.str());
```

Comment:
Can you let me know if there is any reason behind using `engine_id.str()` ?. It doesn't seem to give useful info but is it just to differentiate engines when multiple engines are being loaded ?   
```
tensorrt_engine_mod_0x7fffffffd900
```

Reply:
yeah its so engines dont clash by name. All it is is a "UID"